src/Illuminate/Console/composer.json

```diff
@@ -24,6 +24,7 @@
         "laravel/prompts": "^0.1.12",
         "nunomaduro/termwind": "^2.0",
         "symfony/console": "^7.0",
+        "symfony/polyfill-php83": "^1.28",
```

Comment:
Don't we only need it here? main composer.json and the one from http already require `symfony/http-foundation`

Reply:
That's another option, the only risk if `symfony/http-foundation` suddenly remove the deps, we should be able to detect but any installation using outdated laravel versions with latest symfony deps could cause `symfony/polyfill-php83` to be missing.

Reply:
Right. Will leave it up to you. Just feels like a strange dependency to keep around.